packages/terser/README.md

```diff
@@ -0,0 +1,59 @@
+[npm]: https://img.shields.io/npm/v/@rollup/plugin-terser
+[npm-url]: https://www.npmjs.com/package/@rollup/plugin-terser
+[size]: https://packagephobia.now.sh/badge?p=@rollup/plugin-terser
+[size-url]: https://packagephobia.now.sh/result?p=@rollup/plugin-terser
+
+[![npm][npm]][npm-url]
+[![size][size]][size-url]
+[![libera manifesto](https://img.shields.io/badge/libera-manifesto-lightgrey.svg)](https://liberamanifesto.com)
+
+# @rollup/plugin-terser
+
+🍣 A Rollup plugin to generate a minified output bundle.
+
+## Requirements
+
+This plugin requires an [LTS](https://github.com/nodejs/Release) Node version (v14.0.0+) and Rollup v1.20.0+.
```

Comment:
it actually requires Rollup 2+ according to the peer dependency

Reply:
good point :see_no_evil: